src/webgl/p5.RendererGL.js

```diff
@@ -365,6 +365,9 @@ p5.prototype.setAttributes = function(key, value) {
   this.push();
   this._renderer._resetContext(attr);
   this.pop();
+  if (this._renderer._curCamera) {
+    this._renderer._curCamera._renderer = this._renderer;
+  }
```

Comment:
Great! I would say that it might be a good idea to put a test into `test/unit/webgl/p5.Camera.js`. Something to the effect of:
```
test('Camera is Renderer is correctly set after setAttributes', function() {
      var myCam2 = myp5.createCamera();
      assert.deepEqual(myCam2, myp5._renderer._curCamera);
      myp5.setAttributes('antialias', true);
      assert.deepEqual(myCam2._renderer, myp5._renderer);
    });
```
will probably be good. Though some more general state consistency tests may be in order soon.

Reply:
Will the suite `RendererGL Current Camera` a good place for this test?

Reply:
Yes that sounds good!